.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,18 @@
+name: Documentation Request
+description: Create an issue for missing or incorrect documentation
+title: "[Documentation]: "
+labels: ["T:Docs"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ✰  Thanks for opening an issue! ✰    
+        Before smashing the submit button please review the template.
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Summary
+      description: Tell us where what you would like to see get added to the documentation or if there is an error in the documentation?
```

Comment:
I would put this in the body instead, otherwise it looks a bit weird: https://github.com/cosmos/cosmos-sdk/blob/marko/17244/.github/ISSUE_TEMPLATE/documentation.yml